README.md

```diff
@@ -151,7 +151,7 @@ Let's go back to our vanilla SwiftUI home view and enhance it using the Composab
 ```swift
 import ComposableNavigator
 
-struct HomeView: View {
+struct ExampleView: View {
```

Comment:
I thought this was right based on the comment below the code sample, but it's a little ambiguous. You should probably double check it.

Reply:
I would actually leave as HomeView. The example is picked up later again in the Navigation tree, where HomeView represents HomeScreen objects. :)